docs/app-dev-context/handle-query.md

```diff
@@ -1,3 +1,3 @@
 Queries are handled by your Workflow.
 
-Do not include any logic that causes [Command](/concepts/what-is-a-command) generation within a Query handler (such as executing Activities). as this will lead to unexpected behavior.
+Don’t include any logic that causes [Command](/concepts/what-is-a-command) generation within a Query handler (such as executing Activities). As this will lead to unexpected behavior.
```

Comment:
```suggestion
Don’t include any logic that causes [Command](/concepts/what-is-a-command) generation within a Query handler (such as executing Activities), as this will lead to unexpected behavior.
```